src/components/browse/singleDatasetView/singleDatasetViewTabs/dapTabContents.tsx

```diff
@@ -15,7 +15,8 @@ const DapTabContents = (props: DapTabContentsProps) => {
         <FontAwesomeIcon
           icon={faUsersRays}
           pull="left"
-          className="text-secondary me-3 fs-4"
+          style={{width: "25px", height: "25px", backgroundColor: "rgba(214,95,48,0.2)", padding:"4px"}}
```

Comment:
could you make a 72*72 background for each icon in the open box? And could be bigger icons as well like in the design

Reply:
I can make it bigger but 72x72 looks like this
![image](https://user-images.githubusercontent.com/31940280/183618839-7e74a4e5-b5d0-4da4-ab7d-1ab71af82fc6.png)

Reply:
then you can make icons smaller than background like in the design